### PR TITLE
Add basic large data scenarios

### PR DESCRIPTION
**Pull request by byencho**
This PR adds one basic scenario (with and without Bridge) to the sample app. A "random" `Bitmap` can be generated to simulate a large amount of data and navigation to a different `Activity` is used to show the lack of a crash when using Bridge.

### CODE REVIEW

File: bridgesample/src/main/java/com/livefront/bridgesample/scenario/activity/NonBridgeLargeDataActivity.kt [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        supportActionBar!!.apply {
+            setDisplayHomeAsUpEnabled(true)
+            setHomeButtonEnabled(true)
+            setTitle(R.string.non_bridge_large_data_screen_title)
+        }
+
+        headerText.setText(R.string.non_bridge_large_data_header)
+
+        if (this::bitmap.isInitialized) {
+            // The Bitmap is restored from a saved state. We can just show it directly
```
Comment (weisers) [inline bridgesample/src/main/java/com/livefront/bridgesample/scenario/activity/NonBridgeLargeDataActivity.kt]: Minor -- add a period to the end of the second sentence.

**Comment by weisers** [comment]
Just one minor comment, otherwise this works for me.

File: bridgesample/src/main/java/com/livefront/bridgesample/scenario/activity/NonBridgeLargeDataActivity.kt (hunk 1 above)
Comment (byencho) [reply bridgesample/src/main/java/com/livefront/bridgesample/scenario/activity/NonBridgeLargeDataActivity.kt]: Sure.

**Review by dperez37 — APPROVED** [review]

**Comment by byencho** [comment]
Thanks @dperez37 and @weisers

**Pull request merged**